### PR TITLE
[MOD-113][Fix] Fix date on moderation list page

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose
All dates on the moderation list page were the current date due to a rename mistake.

## Changes
* less than a day show relative time
* update wording on accepted and rejected page to be more relevant

screen shot 2017-09-29 at 11 42 05 am
screen shot 2017-09-29 at 11 42 23 am

### CODE REVIEW

File: app/components/moderation-list-row/component.js [hunk 1]
```diff
@@ -11,7 +32,25 @@ export default Ember.Component.extend({
...
     },
 
+    gtDay: Ember.computed('submission.dateLastTransitioned', function() {
+        return moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
+    }),
+
+    relevantDate: Ember.computed('submission.dateLastTransitioned', 'gtDay', function() {
+        return this.get('gtDay') ?
+            moment(this.get('submission.dateLastTransitioned')).format('MMMM DD, YYYY') :
+            moment(this.get('submission.dateLastTransitioned')).fromNow();
```
Comment (sheriefvt) [inline app/components/moderation-list-row/component.js:42]: I like this change, but concerned on how the senetece generated by fromNow() (e.g. 44 seconds ago) will be translated.

**Review by sheriefvt — CHANGES_REQUESTED** [review]
Looks good 👍 , only one concern/change. Feel free to ignore 🐼  if you want

File: app/components/moderation-list-row/component.js (hunk 1 above)
Comment (sheriefvt) [reply app/components/moderation-list-row/component.js:42]: also, do we need to check with product on wording changes

Comment (laurenbarker) [reply app/components/moderation-list-row/component.js:42]: Oh good point. I hadn't considered that.

Comment (laurenbarker) [reply app/components/moderation-list-row/component.js:42]: Okay, [moment supports internationalization](https://momentjs.com/docs/#/i18n/) so if there are multiple languages I think we would just need to set the language in https://github.com/CenterForOpenScience/ember-osf-reviews/blob/develop/app/routes/application.js.

Comment (laurenbarker) [reply app/components/moderation-list-row/component.js:42]: I'll double check on the wording changes. We couldn't use what was in the mockups due to the difficulty of getting the moderator's name.

Comment (laurenbarker) [reply app/components/moderation-list-row/component.js:42]: Wording approved by product and MOD-115 added to backlog to update the wording with the moderator's name.

**Pull request merged**